actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -91,12 +91,26 @@ module DateHelper
       #   distance_of_time_in_words(from_time, from_time + 50.minutes, scope: 'datetime.distance_in_words.short') # => "an hour"
       #   distance_of_time_in_words(from_time, from_time + 3.hours, scope: 'datetime.distance_in_words.short')    # => "3 hours"
       def distance_of_time_in_words(from_time, to_time = 0, options = {})
+        if from_time.is_a?(Numeric)
+          from_time = Time.at(from_time)
+        elsif from_time.respond_to?(:to_time)
+          from_time = from_time.to_time
+        else
+          raise ArgumentError, "first argument must be convertible to_time"
+        end
+
+        if to_time.is_a?(Numeric)
+          to_time = Time.at(to_time)
+        elsif to_time.respond_to?(:to_time)
+          to_time = to_time.to_time
+        else
+          raise ArgumentError, "second argument must be convertible to_time"
+        end
```

Comment:
I'd love to extract a small, private interface to perform this logic. Something like:
```ruby
def normalize_to_time(value)
  if value.is_a?(Numeric)
    Time.at(value)
  elsif value.responds_to?(:to_time)
    value.to_time
  else
    yield(value) if block_given?
  end
end
```
Then we could replace these two similar blocks of code with something like:
```ruby
from_time = normalize_to_time(from_time) { raise ArgumentError, "first argument must be convertible_to_time" }
to_time = normalize_to_time(to_time) { raise ArgumentError, "second argument must be convertible_to_time" }
```

The only reason that I didn't do that here is I'm not completely certain how Rails uses this module. If I put that in a private method, would it be accessible in the places where this module is included?

Anyway, that feels like it'd be a valuable change, but in the end I decided it might be better to discuss than me stumbling around trying to figure it out.

Thanks for the feedback in helping get this in!